source/agora/consensus/data/Transaction.d

```diff
@@ -247,3 +248,44 @@ public bool getSumOutput (const Transaction tx, ref Amount acc)
             return false;
     return true;
 }
+
+/*******************************************************************************
```

Comment:
This function shouldn't live in this module. The `consensus/data/*` modules should only contain data definitions.

Technically the fees are only for the Validators. So maybe it should live somewhere close to the `Validator.d` module. Perhaps a `Fee.d` module would work.